src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -19,6 +19,12 @@
  * Represents the in-memory model of Anakin data.
  */
 public class ModelManager extends ComponentManager implements Model {
+    /**
+     * This enum encapsulates the different types of sorts that can be performed.
+     */
+    public enum SortingType {
+        PERFORMANCE, ALPHABETICAL
```

Comment:
Same as above, shouldn't it be lexicographical as we permit string contains numbers?

Reply:
Yes, I agree. Thanks for spotting this! I've edited the code accordingly.